### DISCUSSION TITLE
HL2 compatibility.

### DISCUSSION THREAD

**Discussion by mcdent** [ramdor/Thetis#405] [General]
Hi,
I read a few weeks ago that the person who was maintaining the fork of Thetis for HL2 is no longer maintaining. I think there was some talk about the changes for HL2 being included in this version. I hope I read that correctly.
I wonder if things are any nearer with this? Or is this likely not to happen for some time, or at all?

It goes without saying, thank you for the work you have put in to this awesome software up to now, it is very
much appreciated.
Thanks
Mike
G6PHF

**Comment by Roturbo** [comment]
Hello to all,

i have friends running HL2 remote, i think it was some changes in firmware or maybe only on software, 
they have a port to configure and have to also configure the modem for external access with 
specific port to connect to HL2, and that thing is is really fast.

The latency is so small that sometimes i think is less than i have with a dedicated card on my PC
running the Anan 100D, it was really impressive because at start all pointed on the wrong way.

They have other bugs that may never be corrected, but i can´t tell if someone is local or remote,
if the Wi-Fi is good it also runs from small tablets.

**Maybe this is a interesting feature for the old Anan´s**, many users will love to use it in remote once
it is only a question of software coding/configuration.

Regards.
.

**Comment by csylvain** [comment]
see comments - https://github.com/ramdor/Thetis/pull/409
